test/testFramework.js

```diff
@@ -0,0 +1,11 @@
+const checkSame = (actualOutput, expectedOutput) => actualOutput === expectedOutput;
+
+const checkNaN = actualOutput => isNaN(actualOutput);
```

Comment:
don't forget to check for things that are never used, and remove them from all files, including in the test environment